unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/AvatarSlot/AvatarSlotComponentView.cs

```diff
@@ -0,0 +1,108 @@
+using System;
+using TMPro;
+using UnityEngine;
+using UnityEngine.EventSystems;
+using UnityEngine.UI;
+
+namespace DCL.Backpack
+{
+    public class AvatarSlotComponentView : BaseComponentView, IPointerClickHandler, IAvatarSlotComponentView, IComponentModelConfig<AvatarSlotComponentModel>
```

Comment:
You should use the `BaseComponentView<AvatarSlotComponentModel>` approach instead of using `IComponentModelConfig<AvatarSlotComponentModel>`. Take a look to `PassportPlayerInfoComponentView`.

Reply:
Done